docs/fidesops/docs/guides/saas_config.md

```diff
@@ -158,25 +165,62 @@ authentication:
+    - `headers` and `query_params` The HTTP headers and query parameters to include in the request.
+        - `name` the value to use for the header or query param name.
+        - `value` can be a static value or use a `<dynamic_value>` which will be replaced with the value sourced from the `request_param` with a matching name.
+    - `body` (optional) static or dynamic request body, with dynamic portions enclosed in brackets, just like `path`. These dynamic values will be replaced with values from `request_params`. For update requests, you'll need to additionally annotated `<masked_object_fields>` as a placeholder for the Fidesops generated update values.
     - `request_params`
-        - `name` Used as the key for query param values, or to map this param to a value placeholder in the path.
-        - `type` Can be "query", "path", or "body".
+        - `name` Used as the key to reference this value from dynamic values in the path, headers, query, or body params.
         - `references` These are the same as `references` in the Dataset schema. It is used to define the source of the value for the given request_param.
-        - `identity` This denotes the identity value that this request_param should take.
-          - `default_value` Hard-coded default value for a `request_param`. This is most often used for query params since a static path param can just be included in the `path`.
+        - `identity` Used to access the identity values passed into the privacy request such as email or phone number.
+        - `connector_param` Used to access the user-configured secrets for the connection.
     - `data_path`: The expression used to access the collection information from the raw JSON response.
     - `postprocessors` An optional list of response post-processing strategies. We will ignore this for the example scenarios below but an in depth-explanation can be found under [SaaS Post-Processors](saas_postprocessors.md)
-    - `pagination` An optional strategy used to get the next set of results from APIs with resources spanning multiple pages. Details can be found under [SaaS Pagination](saas_pagination.md)
+    - `pagination` An optional strategy used to get the next set of results from APIs with resources spanning multiple pages. Details can be found under [SaaS Pagination](saas_pagination.md).
+
+## Request params in more detail
+The `request_params` list is what provides the values to our various placeholders in the path, headers, query params and body. Values can be `identities` such as email or phone number, `references` to fields in other collections, or `connector_params` which are defined as part of configuring a SaaS connector. Whenever a placeholder is encountered, the placeholder name is looked up in the list of `request_params` and corresponding value is used instead. Here is an example of placeholders being used in various locations:
```

Comment:
Now that `query_params` are separated, the name of `request_params` isn't intuitive without these docs, it's really where we're specifying request values. Just a personal opinion.

Reply:
@eastandwestwind brought up the same point this morning, I'm open to suggestions for a better name, some ideas:

- param_value_map
- param_values
- request_param_values

Reply:
ah good suggestions, I like `param_values`, shorter than `request_param_values`

Reply:
Thanks for all the efforts to change this